lib/tasks/data_migrations.rake

```diff
@@ -0,0 +1,15 @@
+namespace :data_migrations do
+  desc "Set submission type for forms to match the old style submission format"
+  task set_forms_submission_format: :environment do
+    [
+      ["email", []],
+      ["email_with_csv", %w[csv]],
+      ["email_with_csv_and_json", %w[csv json]],
+      ["email_with_json", %w[json]],
+      ["s3", %w[csv]],
+      ["s3_with_json", %w[json]],
+    ].each do |submission_type, submission_format|
+      Form.where(submission_type:).update_all(submission_format:)
```

Comment:
Should we update the form_documents to set the submission_format as part of the same migration?

Reply:
Okay I've added a line to do the form documents as well